apps/nextra/pages/en/build/smart-contracts/digital-asset.mdx

```diff
@@ -2,416 +2,325 @@
-callable directly from off chain. Creators need to write their own modules that
-use these functionalities or use "no code" solutions also provided in the
-framework. One such solution is [aptos_token](#aptos-token) which provides
-functionalities such as custom metadata (via PropertyMap) and soulbound.
-
-## Comparison to the legacy Aptos Token Standard
-
-Digital Asset uses Aptos [objects](https://github.com/aptos-labs/aptos-core/blob/main/aptos-move/framework/aptos-framework/sources/object.move)
-rather than account resources traditionally used in Move. This allows for
-storing data outside the account and adding flexibility in this way.
-
-- Tokens can be easily extended with custom data and functionalities without
-requiring any changes in the framework
-- Transfers are simply a reference update
-- Direct transfer is allowed without an opt-in
-- NFTs can own other NFTs adding easy composability
-- Soul bound tokens can be easily supported
-
-## Collections and tokens as objects
-
-In this Token standard, both collections and tokens will be separate
-[objects](objects.mdx). They have their own distinct addresses and can be
-referenced both on and off chain by address. Each object can contain multiple
-resources so collections and tokens are extensible by default, allowing the
-creator to add custom data and functionalities without having to modify the
-framework.
-
-On chain, another struct can include a reference to the collection or token
-objects like below:
-
-```move
-module 0x42::example {
-  struct ReferenceExample has key {
-    my_collection: Object<Collection>,
-    my_token: Object<Token>,
-  }
-}
-```
+import { Callout } from 'nextra/components'
+import { ThemedImage, GraphQLEditor } from "@components/index";
 
-where both `my_collection` and `my_token` are addresses (with `Object<>` wrapper).
+# Aptos Digital Asset (DA) Standard
 
-Off-chain, the address of the object can be passed along to replace object
-arguments in entry functions called by transaction creation as arguments. For
-example:
+The Digital Asset (DA) standard is a modern Non-Fungible Token (NFT) standard for Aptos. NFTs represent unique assets on-chain, and are stored in collections. These NFTs can be customized to later be transferred, soulbound, burned, mutated, or customized via your own smart contracts.
 
-```move
-module 0x42::example {
-  public entry fun my_function(my_collection: Object<Collection>) {
-    // Do something with the collection
-  }
-}
-```
+This standard replaces the legacy [Aptos Token Standard](aptos-token.mdx). The most important improvements to note are:
+
+| **Improvement** | **Description** |
+|-----------------|-----------------|
+| Token Extension | Tokens can be easily extended since they are implemented using Move [Objects](objects.mdx). |
+| Direct NFT Transfer | You can now directly transfer NFTs without the recipient “opting-in” on-chain. |
+| NFT Composability | NFTs can own other NFTs for easy composability. |
```

Comment:
```suggestion
| **Token Extension** | Tokens can be easily extended since they are implemented using Move [Objects](objects.mdx). |
| **Direct NFT Transfer** | You can now directly transfer NFTs without the recipient “opting-in” on-chain. |
| **NFT Composability** | NFTs can own other NFTs for easy composability. |
```